subsys/net/ip/Kconfig

```diff
@@ -86,8 +86,47 @@ config NET_QEMU_ETHERNET
 	  Connect to host system via Qemu ethernet driver support. One such
 	  driver that Zephyr supports is Intel e1000 ethernet driver.
 
+config NET_QEMU_USER
+	bool "SLIRP"
+	help
+	  Connect to host system via Qemu's built-in User Networking support. This is easy
```

Comment:
this is a bit too dense for a kconfig, I suggest to keep concise and have all the details in some documentation page in the networking chapter.

Reply:
We could have small chapter in doc/guides/networking/networking_with_host.rst about SLIRP, and more detailed detailed information in separate file in that directory.

Reply:
Fixed..